app/celery/scheduled_tasks.py

```diff
@@ -367,3 +369,17 @@ def remove_yesterdays_planned_tests_on_govuk_alerts():
         name=TaskNames.PUBLISH_GOVUK_ALERTS,
         queue=QueueNames.GOVUK_ALERTS
     )
+
+
+@notify_celery.task(name='delete-old-records-from-events-table')
+@cronitor('delete-old-records-from-events-table')
+def delete_old_records_from_events_table():
+    delete_events_before = datetime.utcnow() - timedelta(weeks=52)
+    event_query = Event.query.filter(Event.created_at < delete_events_before)
+
+    current_app.logger.info(
+        f"Preparing to delete {event_query.count()} historical events from before {delete_events_before}."
+    )
```

Comment:
`delete()` returns the row count so we can save the extra query cycle with something like

```suggestion
    deleted_count = event_query.delete()
    current_app.logger.info(
        f"Deleted {deleted_count} historical events from before {delete_events_before}."
    )
```